src/Sarif.FunctionalTests/ConverterTestData/StaticDriverVerifier/checkadddevice_bug1.tt.sarif

```diff
@@ -32,154 +32,105 @@
             {
               "locations": [
                 {
+                  "fullyQualifiedLogicalName": "OS",
                   "kind": "call",
+                  "callee": "main",
                   "importance": "unimportant",
                   "properties": {
-                    "State": "^",
-                    "Caller": "OS",
-                    "Callee": "main"
+                    "currentDataValues": "^"
                   }
                 },
                 {
                   "step": 1,
-                  "message": "====Auto=====",
                   "kind": "continuation",
-                  "importance": "unimportant",
-                  "properties": {
-                    "State": "^====Auto====="
-                  }
+                  "importance": "unimportant"
                 },
                 {
                   "step": 2,
-                  "message": "====Auto=====,====Pre=====,____true",
+                  "fullyQualifiedLogicalName": "main",
                   "kind": "call",
+                  "callee": "_sdv_init4",
                   "importance": "unimportant",
                   "properties": {
-                    "State": "^====Auto=====^====Pre=====^____true",
-                    "Caller": "main",
-                    "Callee": "_sdv_init4"
+                    "currentDataValues": "^====Pre=====^____true"
                   }
                 },
                 {
                   "step": 3,
-                  "message": "====Auto=====",
                   "kind": "continuation",
-                  "importance": "unimportant",
-                  "properties": {
-                    "State": "^====Auto====="
-                  }
+                  "importance": "unimportant"
                 },
                 {
                   "step": 4,
-                  "message": "====Auto=====",
                   "kind": "assignment",
-                  "importance": "unimportant",
-                  "properties": {
-                    "State": "^====Auto====="
-                  }
+                  "importance": "unimportant"
                 },
                 {
                   "step": 5,
-                  "message": "====Auto=====",
                   "kind": "assignment",
-                  "importance": "unimportant",
-                  "properties": {
-                    "State": "^====Auto====="
-                  }
+                  "importance": "unimportant"
                 },
                 {
                   "step": 6,
-                  "message": "====Auto=====",
+                  "fullyQualifiedLogicalName": "main",
```

Comment:
Is this right? In Step 0, `OS` called `main`. In Step 2, `main` called `_sdv_init4`. Now in Step 6 we are returning from `_sdv_init4` to `main`, so `fullyQualifiedLogicalName` -- which refers to the code that's currently executing -- should be `_sdv_init4`.

Now we know we're going to add some more properties to `annotatedCallLocation`, such as a property that specifies the target of the return when the `kind` is `callReturn` (see Issue sarif-standard/sarif-spec#249, which I filed this morning). That's the place that "`main`" would go.